### ISSUE TITLE
Why we should do unfold？

### ISSUE THREAD

**Issue by haochange** [sephirothhua/Lite_Upsample_Network_Carafe-unoffical-#2]
x_mat = torch.nn.functional.unfold(x, kernel_size=self.kernel_up, padding=r, stride=1)
